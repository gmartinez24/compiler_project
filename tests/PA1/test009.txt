// Grant Muirheid
// Testing that the parser can separate integers from identifiers

main

int myvar1 = 5;;
int 2myvar = 3a;

bool a = myvar1 + 2myvar;