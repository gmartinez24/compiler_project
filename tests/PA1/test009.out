MAIN
INT
IDENT   myvar1
ASSIGN
INT_VAL   5
SEMICOLON
SEMICOLON
INT
INT_VAL   2
IDENT   myvar
ASSIGN
INT_VAL   3
IDENT   a
SEMICOLON
BOOL
IDENT   a
ASSIGN
IDENT   myvar1
ADD
INT_VAL   2
IDENT   myvar
SEMICOLON
EOF